prowler/providers/m365/services/teams/teams_service.py

```diff
@@ -75,6 +77,9 @@ def _get_global_meeting_policy(self):
                     designated_presenter_role_mode=global_meeting_policy.get(
                         "DesignatedPresenterRoleMode", "EveryoneUserOverride"
                     ),
+                    allow_security_end_user_reporting=global_meeting_policy.get(
+                        "AllowSecurityEndUserReporting", True
```

Comment:
```suggestion
                        "AllowSecurityEndUserReporting", False
```
I would set the non compliant value if we couldn't retrieve the actual one.

Reply:
I say this because, from my point of view, it's better to create a false positive finding than a false negative.

Reply:
yes, agree 👍